pytransform3d/transform_manager/_utils.py

```diff
@@ -0,0 +1,23 @@
+import numpy as np
+
+from .. import transformations as pt
+
+
+def find_two_closest_indices(arr, target_value):
```

Comment:
This functionality will be needed on any array-based time representation - so I offloaded that ...